alembic/versions/2c4af13eef71_migration_message.py

```diff
@@ -0,0 +1,198 @@
+"""Migration message
```

Comment:
Since you are modifying just role and user models, it is not necessary to run fresh migration for the entire database hence these migration files should not contain migration for the entire database. Also, kindly have only one migration file in your PR. Please work with the backend team to guide you through this. Please review other team member's PRs to avoid having the same slips in your PR.

Reply:
Thank you for pointing these issues out @Bonifase  I will endeavor to do as advised.